frame/contracts/src/exec.rs

```diff
@@ -1349,16 +1360,7 @@ where
 				msg.len().saturating_sub(capacity).min(buffer.len())
 			};
 			buffer.drain(0..num_drain);
-			buffer
-				.try_extend(&mut msg)
-				.map_err(|_| {
-					log::debug!(
-						target: "runtime::contracts",
-						"Debug message to big (size={}) for debug buffer (bound={})",
-						msg.len(), DebugBufferVec::<T>::bound(),
-					);
-				})
-				.ok();
+			buffer.try_extend(&mut msg).ok();
```

Comment:
What if the error message is bigger than the debug buffer? I guess we want to enforce a minimum size in the `integrity_check`.

Reply:
The error message in question is that one defined few lines before: `"Debug message to big (size={}) for debug buffer (bound={})"`, it's 59 bytes. I added the integrity check enforcing `256` bytes minimum just in case.